src/imgix-core-js.js

```diff
@@ -124,6 +146,56 @@
       }
     };
 
+    ImgixClient.prototype.buildSrcSet = function (path, params) {
+      var width = params ? params['w'] : undefined;
+      var height = params ? params['h'] : undefined;
+      var aspectRatio = params ? params['ar'] : undefined;
+      if (params) {
+        delete params.dpr;
```

Comment:
To the best of my knowledge, this would be the correct way to delete a particular key. Just want to be sure that there isn't some sort of 'gotcha' to doing it this way.

Reply:
Nope, this is the way to do it.